### PR TITLE
DOC-763: Add notification / editor blocking behaviour to the 5.7 release notes

### PR DESCRIPTION
**Pull request by ashfordneil**
Related Ticket: DOC-763

Description of Changes:
* Document the TINY-6686 improvements

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- ~[ ] `_data/nav.yml` has been updated (if applicable)~
- ~[ ] Files has been included where required (if applicable)~
- ~[ ] Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes57.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Notifications are now shown when the editor is blocked
+
+Notifications opened with the Notifications Api (`NotificatonApi`) will remain visible when the editor's progress state is set to true using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to true. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```
Comment (JamesToohey) [inline release-notes/release-notes57.md:43]:
```suggestion
Notifications opened with the Notifications API (`NotificationApi`) will remain visible when the editor's progress state is set to `true` using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate). Additionally, popups within the editor will be closed while the editor's progress state is set to `true`. For information on the Notifications API see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

Should it also be ```(`NotificationAPI`)```?

File: release-notes/release-notes57.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Notifications are now shown when the editor is blocked
```
Comment (lnewson) [inline release-notes/release-notes57.md:41]:
```suggestion
### Changes to notifications and popups when the editor is blocked
```
This is not about notifications, it's about the changes that happen when the progress state is set to true. So I've just reworded it to reflect that.

File: release-notes/release-notes57.md (hunk 1 above)
Comment (lnewson) [inline release-notes/release-notes57.md:43]:
```suggestion
Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to true using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will be closed while the editor's progress state is set to true.

For information on the editor `setProgress()` API, see [Editor API - `setProgressState`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).

For information on the Notifications API, see [Create custom notifications]({{ site.baseurl }}/advanced/creating-custom-notifications).
```

This needs breaking up a little and we shouldn't mention `NotificationApi` as that's confusing. I did originally work out that part in the brackets was referring to the type, but types aren't documented and isn't something you'd find in a sentence normally anyways.

**Review by lnewson — CHANGES_REQUESTED** [review]

File: release-notes/release-notes57.md [hunk 3]
```diff
@@ -38,6 +38,14 @@ For information on the `table_column_resizing` option, see: [Table plugin - `tab
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Changes to notifications and popups when the editor is blocked
+
+Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to `true` using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will now be closed while the editor's progress state is set to `true`.
```
Comment (metricjs) [inline release-notes/release-notes57.md:43]: considering we had to clarify amongst ourselves what popups were, maybe some examples would be good? not sure. I don't know if/where we use "popups" as a term elsewhere.

**Review by metricjs — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

File: release-notes/release-notes57.md (hunk 3 above)
Comment (ashfordneil) [reply release-notes/release-notes57.md:43]: We did consider adding a "For information about popups see ..." but then realised there was nowhere in the docs that we explained popups as a concept. Do you think we should Jira up adding that (and then updating this release note)?

File: release-notes/release-notes57.md [hunk 4]
```diff
@@ -38,6 +38,14 @@ For information on the `table_column_resizing` option, see: [Table plugin - `tab
...
+
+For information on the {{ site.productname }} `setProgress()` API, see [Editor API - `setProgressState`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).
```
Comment (tylerkelly13) [inline release-notes/release-notes57.md:45]: Restruct the "for infos" pls

For information on:

- thing one
- thing two

File: release-notes/release-notes57.md (hunk 3 above)
Comment (tylerkelly13) [reply release-notes/release-notes57.md:43]: maybe 

> popups within the editor will ...

to 

> popups within the editor (such as thing popup and otherthing popup) will ...

(obviously replacing thing and other thing) (reminder: don't say- eg, ie, or etc)

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**